tsconfig.json

```diff
@@ -1,22 +1,27 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
     "outDir": "build",
-    "target": "esnext",
+    "target": "es5",
```

Comment:
ES2017 is supported by all major browsers. I don't think you need to go as low as ES5.

Reply:
Alright I'll change this to ES2017